doc/man1/openssl-s_server.pod.in

```diff
@@ -396,9 +384,6 @@ with B<-early_data>.
 =item B<-tlsextdebug>
 
 Print a hex dump of any TLS extensions received from the server.
-
```

Comment:
Stray deletion?

Reply:
no, rebase mistake; the lines that follow should have been part of the -www flag.  fixup commit pushed.  good find!